src/plugins/irc-events/topic.js

```diff
@@ -9,11 +9,9 @@ module.exports = function(irc, network) {
 			return;
 		}
 		var from = data.nick || chan.name;
-		var self = false;
-		if (from.toLowerCase() == irc.me.toLowerCase()) {
-			self = true;
-		}
+		var self = (from.toLowerCase() == irc.me.toLowerCase());
```

Comment:
Hey @kalaksi, really sorry if you took it that way, I wasn't trying to be mean or anything, just humorous. I would never do that to someone who nicely contributed! :-)
So yes, I am always in favor of improving the code. Would be great if you can apply my suggestion! Thanks for spotting that.